bindings/jupyter/setup.py

```diff
@@ -94,24 +91,25 @@ def run(self):
 setup(name='mlvis',
       cmdclass={'develop': develop,
                 'init_jrequirements': init_jrequirements_command},
-      version='0.0.8',
+      version='0.1.0',
       description='A wrapper around react components for use in jupyter notebooks',
       long_description='{}'.format(read(os.path.join('docs', 'introduction.md'))),
       long_description_content_type='text/markdown',
       keywords=['data', 'visualization', 'machine learning'],
       url='https://github.com/uber/manifold.git',
       author='Hong Wang',
-      author_email='hongw@mlvis.com',
+      author_email='hongw@uber.com',
       license='Apache License 2.0',
       include_package_data=True,
       packages=find_packages(),
       zip_safe=False,
+      package_data={'': ['mlvis/jrequirements.json']},
```

Comment:
Do we need a package key here?

Reply:
Nope, because we use the default package_dir setting, the root of the package is mlvis/